test/gtest/ucp/ucp_test.cc

```diff
@@ -445,7 +460,8 @@ void ucp_test::modify_config(const std::string& name, const std::string& value,
 {
     ucs_status_t status;
 
-    status = ucp_config_modify(m_ucp_config, name.c_str(), value.c_str());
+    status = ucs_config_parser_set_value(m_ucp_config, m_ucp_cfg_table,
+                                         name.c_str(), value.c_str());
```

Comment:
which gtest for example fail if we don't do this?
instead, can we remove m_ucp_cfg_table and add helper function to ucp:  `ucp_config_modify_internal(...)`
This function will NOT try to save UCT cached configuration, and we will this function from `ucp_config_modify` and from gtest (here)

Reply:
I'll add ```ucp_config_modify_internal``` API

Reply:
> which gtest for example fail if we don't do this?

@yosefe Some gtest cases will fail if we don't do this
```
althca on worker-xxx 
./test/gtest/gtest --gtest_filter=tcp/test_ucp_tag_stats.rndv_expected/0
./test/gtest/gtest --gtest_filter=shm_ib/test_ucp_tag_stats.eager_unexpected/0
./test/gtest/gtest --gtest_filter=self/test_ucp_tag_stats.eager_expected/0
./test/gtest/gtest --gtest_filter=tcp/test_ucp_tag_stats.sync_unexpected/0

docker/worker
./test/gtest/gtest --gtest_filter=shm_ib/test_ucp_tag_stats.sync_expected/0
./test/gtest/gtest --gtest_filter=tcp/test_ucp_tag_stats.sync_unexpected/0
./test/gtest/gtest --gtest_filter=self/test_ucp_tag_stats.sync_expected/0
./test/gtest/gtest --gtest_filter=self/test_ucp_tag_stats.sync_unexpected/0

hwi/worker
./test/gtest/gtest --gtest_filter=tcp/test_ucp_tag_stats.eager_expected/0
./test/gtest/gtest --gtest_filter=dcx/test_ucp_tag_offload_stats.rndv/0
./test/gtest/gtest --gtest_filter=ud/test_ucp_tag_stats.sync_expected/0
./test/gtest/gtest --gtest_filter=rcx/test_ucp_tag_offload_stats.rndv/0
```